packages/api/src/api/users.ts

```diff
@@ -6,6 +6,7 @@ export const users = Router();
 users.use(express.json());
```

Comment:
If this does't break tests, can you remove this? It shouldn't have been included the first time around but we missed it
```suggestion
```